docs/targets/datadog.md

```diff
@@ -36,49 +36,57 @@ For more information about using Datadog, please refer to the [Datadog documenta
 
 A Datadog event Target accepts the following [CloudEvent][ce] types:
 
-### io.triggermesh.datadog.event
+### io.triggermesh.datadog.event.post
 
-Events of this type are alert metrics to be published to Datadog.
+Events of this type contain event messages to be published to Datadog.
 
 This type expects a [JSON][ce-jsonformat] payload with the following properties:
```

Comment:
Do you have the replacement properties/values?

Reply:
what are replacement properties?

Reply:
Okay nevermind, the diff rendering made it appear that `io.triggermesh.datadog.metric` was the property type for the next batch instead of the table being a continuation of `io.triggermesh.datadog.submit`.